nemo/collections/common/metrics/punct_er.py

```diff
@@ -0,0 +1,473 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
The first line should be:
`# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.`

Reply:
Thank you, fixed